MKAssignment3/src/mkassignment3/MKAssignment3.java

```diff
@@ -0,0 +1,31 @@
+
+package mkassignment3;
+/**
```

Comment:
Good work commenting your code.  I forgot to do this to mine for this assignment.